src/CMakeLists.txt

```diff
@@ -63,28 +63,11 @@ endif()
 include(${PROJECT_SOURCE_DIR}/cmake/TargetLinkLibrariesSystem.cmake)
 
 include_directories(
-    ${PROJECT_SOURCE_DIR}/third_party/googletest/googletest/include
-    ${PROJECT_SOURCE_DIR}/third_party/sql-parser/src
-    ${PROJECT_SOURCE_DIR}/third_party/magic_enum/include
-
     ${PROJECT_SOURCE_DIR}/src/benchmarklib/
     ${PROJECT_SOURCE_DIR}/src/lib/
     ${PROJECT_SOURCE_DIR}/src/plugins/
 )
 
-# Include these libraries as system libraries to silence some warnings caused by -Weverything
-include_directories(
-    SYSTEM
-    ${PROJECT_SOURCE_DIR}/third_party/benchmark/include
-    ${PROJECT_SOURCE_DIR}/third_party/cpp-btree/include/btree
-    ${PROJECT_SOURCE_DIR}/third_party/cxxopts/include
-    ${PROJECT_SOURCE_DIR}/third_party/flat_hash_map
-    ${PROJECT_SOURCE_DIR}/third_party/json
-    ${PROJECT_SOURCE_DIR}/third_party/lz4
-    ${PROJECT_SOURCE_DIR}/third_party/zstd
-    ${PROJECT_SOURCE_DIR}/third_party/tpcds-result-reproduction
```

Comment:
Can we remove `tpcds-result-reproduction` without any replacement because we don't use it for building anything but just use the result table files from it?

Reply:
Removing the include directory here does not remove the git submodule.
```
[submodule "resources/benchmark/tpcds/tpcds-result-reproduction"]
	path = resources/benchmark/tpcds/tpcds-result-reproduction
	url = https://github.com/hyrise-mp/tpcds-result-reproduction.git
```
Thus, after updating the submodules, the result files are available and can be used for query verification.

Reply:
> Removing the include directory here does not remove the git submodule.

Of course not. So the answer to above's question is _yes_.